src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -198,6 +198,12 @@ private void ProviderComboBox_SelectionChanged(object sender, SelectionChangedEv
 
         private void InstallButton_Clicked(object sender, RoutedEventArgs e)
         {
+            // When "Include all files" option is checked, we don't want to write out the files to libman.json
+            if (IncludeAllLibraryFilesRb.IsChecked.Value)
```

Comment:
>IncludeAllLibraryFilesRb.IsChecked.Value [](start = 16, length = 40)

Issues discussed in person:
1) What to do if all selected and checkbox not selected
2) Why files: [] instead of not having it?
3) Sort order seems wonky in dialog

Reply:
As discussed offline, leaving [1] as is.
Fixed 2 in iteration 2.

Justin/me will log bug for 3.

---
In reply to: [197885065](https://github.com/aspnet/LibraryManager/pull/248#discussion_r197885065) [](ancestors = 197885065)